test/unit/arena_decay.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-export MALLOC_CONF="dirty_decay_ms:1000,muzzy_decay_ms:1000,tcache_max:1024"
+export MALLOC_CONF="dirty_decay_ms:1000,muzzy_decay_ms:1000,tcache_max:1024,tcache_gc_incr_bytes:939524096"
```

Comment:
the arena_decay test is overriding nstime_update which affects the GC timer, disable GC for this test.